_inc/client/main.jsx

```diff
@@ -92,6 +92,7 @@ class Main extends React.Component {
 		const tracksUser = this.props.tracksUserData;
 		if ( tracksUser ) {
 			analytics.initialize(
+				tracksUser.blogid,
```

Comment:
I'm seeing at least [1 more location](https://github.com/Automattic/jetpack/blob/master/_inc/idc-notice.js#L21) where we initialize the tracks client, and we haven't updated that to reflect the new parameter order. This will break things.

Reply:
true, my bad

Reply:
Solved as per remarks above and moving blogid to superProps.

Reply:
moved to supeProps